### PR TITLE
VAS-6

### PR DESCRIPTION
**Pull request by vmartos**

### CODE REVIEW

**Comment by aoller** [comment]
👍

File: VAS.Services/PlayerController.cs [hunk 1]
```diff
@@ -445,6 +445,7 @@ protected virtual bool PlaylistSeek (Time time, bool accurate = false, bool sync
 					Log.Debug (String.Format ("There is no playlist element at {0}.", time));
 					return false;
 				}
+				LoadedPlaylist.SetActive (elementAtTime);
```
Comment (ylatuya) [inline VAS.Services/PlayerController.cs]: Shouldn't this be done in LoadPlaylistEvent for every event loaded in the playlist ?

File: VAS.DB/DocumentsSerializer.cs [hunk 2]
```diff
@@ -323,6 +323,9 @@ static string GetDocumentType (IStorable storable)
 		{
 			Type type = storable.GetType ();
 
+			if (DocumentTypeBaseTypes.ContainsValue (type.Name))
+				return type.Name;
```
Comment (ylatuya) [inline VAS.DB/DocumentsSerializer.cs]: This check is wrong, it should be:
if (DocumentTypeBaseTypes.ContainsKey (type))
return DocumentTypeBaseTypes[type]

**Comment by vmartos** [comment]
Ready to codereview again

**Comment by aoller** [comment]
👍

**Pull request merged**